spec/features/idv/end_to_end_idv_spec.rb

```diff
@@ -87,69 +87,76 @@
     validate_return_to_sp
   end
 
-  context 'with an sp that allows in person proofing' do
-    before do
-      allow(IdentityConfig.store).to receive(:in_person_proofing_enabled).and_return(true)
+  scenario 'Verify by mail' do
+    visit_idp_from_sp_with_ial2(sp)
+    user = sign_up_and_2fa_ial1_user
+    complete_all_doc_auth_steps
 
-      ServiceProvider.find_by(issuer: service_provider_issuer(sp)).
-        update(in_person_proofing_enabled: true)
-    end
+    enter_gpo_flow
+    test_go_back_from_request_letter
+    complete_request_letter
+    complete_enter_password_step(user)
 
-    context 'when gpo is allowed for verify by mail' do
-      before do
-        allow(IdentityConfig.store).to receive(:no_verify_by_mail_for_biometric_comparison_enabled).
-          and_return(false)
-      end
+    try_to_go_back_from_letter_enqueued
+    validate_letter_enqueued_page
+    complete_letter_enqueued
+    validate_return_to_sp
+
+    visit sign_out_url
+    user.reload
 
-      scenario 'In person proofing verify by mail', allow_browser_log: true do
-        visit_idp_from_sp_with_ial2(sp)
-        user = sign_up_and_2fa_ial1_user
+    visit_idp_from_sp_with_ial2(sp)
 
-        begin_in_person_proofing
-        complete_all_in_person_proofing_steps(user)
-        test_restart_in_person_flow(user)
+    sign_in_live_with_2fa(user)
 
-        enter_gpo_flow
-        test_go_back_from_request_letter
-        complete_request_letter
+    complete_gpo_verification(user)
+    expect(user.identity_verified?).to be(true)
 
-        test_go_back_in_person_flow
-        complete_enter_password_step(user)
+    acknowledge_and_confirm_personal_key
+    validate_idv_completed_page(user)
+    click_agree_and_continue
 
-        try_to_go_back_from_letter_enqueued
-        validate_letter_enqueued_page
-        complete_letter_enqueued
-        validate_return_to_sp
+    validate_return_to_sp
+  end
 
-        visit sign_out_url
-        user.reload
+  context 'with an sp that allows in person proofing' do
```

Comment:
This spec used to test in-person and GPO in one pass. That is no longer possible so I broke GPO and in-person out into 2 examples.